src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -194,7 +194,7 @@ def compute_snrs(
         A SortingAnalyzer object.
     peak_sign : "neg" | "pos" | "both", default: "neg"
         The sign of the template to compute best channels.
-    peak_mode : "extremum" | "at_index", default: "extremum"
+    peak_mode : "extremum" | "at_index", "peak_to_peak", default: "extremum"
```

Comment:
```suggestion
    peak_mode : "extremum" | "at_index" | "peak_to_peak", default: "extremum"
```

Completely aesthetic :)

Reply:
But important !!
Well spotted :)